test/unit/mongo_logger.test.ts

```diff
@@ -114,6 +115,61 @@ describe('class MongoLogger', async function () {
         expect(buffer).to.have.lengthOf(1);
       });
     });
+
+    describe('willLog', function () {
```

Comment:
If we refactor as my suggestions, then this becomes a unit test of the `willLog()` method and the test no longer needs to know about the internal structure of the severities. I think it makes the tests more easy to reason about here.